strings/french.json

```diff
@@ -84,7 +104,14 @@
   "Open Prompt": "Open Prompt",
   "Leave {channelName}": "Leave {channelName}",
   "Next distribution date is on {date}. Make sure to update your {r1}income details{r2} by then.": "Next distribution date is on {date}. Make sure to update your {r1}income details{r2} by then.",
-  "Date": "Date",
+  "ALL": "ALL",
```

Comment:
If you're going to update the strings, please also use [the latest version](https://github.com/okTurtles/strings/releases/) of Simon's strings utility. It has be improved significantly, and one of the changes is that all of these will be sorted alphabetically.

Reply:
Oh. I missed it... Okay.